mangaki/mangaki/utils/mal.py

```diff
@@ -345,18 +345,27 @@ def insert_into_mangaki_database_from_mal(mal_entries: List[MALEntry],
                 and title.upper() in list(map(str.upper, titles))):
                 first_matching_work = work
 
+            # FIXME: will create many redundant queries.
+            simple_lang = Language.objects.get(code='simple')
```

Comment:
I don't know what kind of `ExtLanguage` should we create for MAL.
MAL does not have explicit lang on his markup, he'll only return:

- title of unknown language
- english title when possible
- synonyms of unknown language

AFAIK.